fs/zbd_zenfs.cc

```diff
@@ -174,6 +174,20 @@ Zone *ZonedBlockDevice::GetIOZone(uint64_t offset) {
   return nullptr;
 }
 
+std::vector<Zone *> ZonedBlockDevice::GetReclaimZones(void) {
+  std::vector<Zone *> GCReclaimZones;
+  for (const auto z : io_zones) {
+    if ((!z->IsUsed()) && (!z->IsFull())) continue;
+    /* once the valid data in a zone is less than 10%, it will be seleted as GC
+     * zone */
+    if ((z->used_capacity_ * ZENFS_GC_THREASHOLD) / z->max_capacity_ <= 1) {
```

Comment:
From my perspective, GC is not called very frequently, and we might need to dynamically configure the threshold. So I would prefer to simply use double multiplication.